react-basic-practice/src/components/OptionsMenu/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import {
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+} from "@chakra-ui/react";
+import { OptionIcon, ViewIcon, EditIcon, DeleteIcon } from "@icons";
+
+interface IconOptionWithMenuProps {
```

Comment:
Pls change naming to `IOptionMenuProps`. We have the format rule for interface.

Reply:
i done

Reply:
Btw, do we need to move this function to foder type.ts

Reply:
Yes, pls follow by previous components